packages/core/src/testE2E/amazonq/featureDev.test.ts

```diff
@@ -212,10 +208,6 @@ describe('Amazon Q Feature Dev', function () {
                 })
             })
 
-            describe('Clicks accept code', () => {
-                insertCodeTests()
```

Comment:
question: even if they are being removed is the flow still tested?

Reply:
No, I think this change also removes about half the tests

Reply:
Could we instead of removing, fix the tests somehow?

Reply:
I added them back as they started to pass once I increased  `waitIntervalInMs`, `waitTimeoutInMs` times